### ISSUE TITLE
Can we use losses other than rate-based losses? 

### ISSUE THREAD

**Issue by PastelBelem8** [google-research/tensorflow_constrained_optimization#11]
It seems that the examples in the repository only use the `error_rate` as the objective function. Would we have reasonable results if we used other out-of-the-box loss functions supported by TensorFlow (e.g., binary cross entropy, or MSE)?

**Comment by AiliAili** [comment]
Hi @PastelBelem8, 
Have you found out how to use the binary cross entropy in your case?

Cheers,
